cpp/examples/ode_secir_ageres.cpp

```diff
@@ -79,38 +81,49 @@ int main()
         params.get<mio::osecir::RiskOfInfectionFromSymptomatic>()[i]    = 0.25;
         params.get<mio::osecir::MaxRiskOfInfectionFromSymptomatic>()[i] = 0.45;
         params.get<mio::osecir::SeverePerInfectedSymptoms>()[i]         = 0.2;
-        params.get<mio::osecir::CriticalPerSevere>()[i]                 = 0.25;
+        params.get<mio::osecir::CriticalPerSevere>()[i]                 = 0.45;
         params.get<mio::osecir::DeathsPerCritical>()[i]                 = 0.3;
     }
 
     model.apply_constraints();
 
-    mio::ContactMatrixGroup& contact_matrix = params.get<mio::osecir::ContactPatterns>();
-    contact_matrix[0] =
-        mio::ContactMatrix(Eigen::MatrixXd::Constant((size_t)nb_groups, (size_t)nb_groups, fact * cont_freq));
-    contact_matrix.add_damping(Eigen::MatrixXd::Constant((size_t)nb_groups, (size_t)nb_groups, 0.7),
-                               mio::SimulationTime(30.));
-
-    mio::TimeSeries<double> secir = simulate(t0, tmax, dt, model);
-
-    bool print_to_terminal = true;
-
-    if (print_to_terminal) {
+    // tests
```

Comment:
hi @HenrZu, the previous example was actually to show how to build a simple age resolved model with three age groups. This is now another example. Can you rather make a new one?

Reply:
Yes, this example was just used to benchmark the extrapolation function. Will reset this before requesting review